channels/pkg/channels/addon.go

```diff
@@ -97,14 +97,16 @@ func (a *Addon) GetRequiredUpdates(k8sClient kubernetes.Interface) (*AddonUpdate
 		return nil, err
 	}
 
-	if existingVersion != nil && !newVersion.replaces(existingVersion) {
+	update, replace := newVersion.updates(existingVersion)
```

Comment:
It looks like `GetInstalledVersion` can return a nil `existingVersion` which I'm guessing is the case for brand new clusters that are having their channels applied for the very first time. If `existingVersion` is nil that would trigger a panic on line 112 of `channel_version.go`

Reply:
I see thanks.